setup.py

```diff
@@ -17,7 +17,7 @@ def readme():
     url="https://github.com/plumerai/larq-zoo",
     packages=find_packages(),
     license="Apache 2.0",
-    install_requires=["numpy~=1.15", "larq~=0.5", "zookeeper>=0.3.1,<0.5.0"],
+    install_requires=["numpy~=1.15", "larq~=0.5", "zookeeper~=0.5"],
```

Comment:
Doesn't it require zookeeper 0.5? If so I would use >=0.5.

Reply:
It does, I have implemented your fix in 79194172937878737ada9144a6c6dd29895061bb

Reply:
We should keep the dependency fixed to `zookeeper~=0.5` to prevent us from accidentally pulling in breaking changes.
For reference `~=0.5` is equivalent to `>=0.5.0,<0.6.0`

Reply:
Reset to `~=0.5`